metabase/metabase
# [FE] Disable broken card refs

Fixes https://github.com/metabase/metabase/issues/29763

With viz settings migrated from field refs keys, we can now use correct field refs for cards. This is required for https://github.com/metabase/metabase/pull/46555. 

There are no behavior changes, but internally field refs in `dataset_query` become different. 

How to verify:
- New -> Question -> Orders -> Save as Q1
- New -> Question -> Q1 -> Open the data source field picker and toggle off some of the fields -> Save
- Check that the field refs in `dataset_query.query.fields` are name-based (e.g. `["field", "ID", {"base-type": ...}]`
